src/tools/rust-analyzer/crates/hir-ty/src/display.rs

```diff
@@ -2,10 +2,7 @@
 //! HIR back into source code, and just displaying them for debugging/testing
 //! purposes.
 
-use std::{
-    fmt::{self, Debug},
-    mem::size_of,
-};
```

Comment:
r-a builds against stable out of tree so this will fail to build when synced back

Reply:
@Veykril I'll switch it to a temporary `allow` instead.

Reply:
Does the same reasoning apply to clippy and miri? I notice that files in those directories are all `allow`ed.